conformance/src/fastify/conformanceApiPlugin.ts

```diff
@@ -905,7 +905,7 @@ export interface ICreateWidgetResponse {
 /** Request for GetWidget. */
 export interface IGetWidgetRequest {
   /** The widget ID. */
-  id?: number;
+  id: number;
```

Comment:
Same as [this comment](https://github.com/heymb/FacilityJavaScript/pull/1/files#r1702268050) if based on same FSD file which it _seems_ to be.